build.rs

```diff
@@ -90,6 +91,7 @@ error: the `wasm32-wasi` target is not installed
     cargo_build(RUST_HTTP_VAULT_CONFIG_TEST);
     cargo_build(RUST_OUTBOUND_REDIS_INTEGRATION_TEST);
     cargo_build(TIMER_TRIGGER_INTEGRATION_TEST);
+    cargo_build(WASI_HTTP_INTEGRATION_TEST);
```

Comment:
This is fine for now since it's following an existing pattern, but I just want to note we should move this out of the build script and into the tests like we did for spin-componentize.